modules/ROOT/pages/deploying-to-multiple-environments.adoc

```diff
@@ -0,0 +1,234 @@
+
+. Double-click each of these new files to open them in new tabs in Studio. By editing them, you can configure the properties of the environment that correspond to the filename. For example, you may wish to add the properties as per the following images. Keep in mind that you can use these properties anywhere in your application.
++
+image::deploying-to-multiple-environments-02.png[]
++
+image::deploying-to-multiple-environments-03.png[]
+
+. Close the properties files.
+
+. Click in the Configuration XML tab of your Studio application, below the canvas.
+
+. Add a *configuration-properties* element to the XML file and let the file name configurable.
+
+[source,xml,linenums]
+----
+<configuration-properties doc:name="Configuration properties" file="${env}.yaml" />
+----
+
+[TIP]
+====
+Learn more about xref:configuring-properties.adoc[configuring variable properties].
+====
+
+[start=7]
+. Add a *Global Property* element to set the default environment name.
+
+[source,xml,linenums]
+----
+<global-property doc:name="Global Property" name="env" value="qa" />
+----
+[start=8]
+. You can change the environment by editing the configured *Global Property* value.
+
+
+== Testing Configured Environment Properties in Studio
+
+You can test that your application uses the specific environment you configured in your variables properties, by running the application in Mule embedded in Studio.
+
+. In the Package Explorer view, right-click the filename of the project you wish to deploy, then select *Run As* > *Mule Application*.
+
+. Studio automatically deploys your application according to the environment variable you specified in the *Global Property* from the previous procedure.
+
+. Again in the Package Explorer view, right-click the filename of the project you wish to deploy, then select *Run As* > *Run Configurations...*.
++
+image::deploying-to-multiple-environments-07.png[]
+
+. Define the System Property in the `Arguments` tab.
++
+image::deploying-to-multiple-environments-08.png[]
+
+. Click the `Run` button to complete the procedure. The properties configured in `prod.yaml` will be used instead of the ones defined in `qa.yaml`.
+
+== Deploying Mule Apps using your Specific Environment Configurations
+
+Mule loads the environment variables when it starts, so depending on your deployment target (CloudHub or On-premises) you need to set the environment when you deploy the app (CloudHub) or when you start your Mule instance (On-premises).
+
+=== Setting a Specific Environment when Starting Mule On-Premises
+
+==== Using the command line
+
+Identify the environment in which to deploy your application at runtime with an environment variable. Execute the command to run Mule in your environment as per the following example:
+
+[source,console,linenums]
+----
+$ mule -M-Dmule.env=prod
+----
+
+Once Mule starts with your configured environment, you can deploy the application with your desired deployment method.
```

Comment:
Once > After

However, what does this mean? If it starts using the configured environment, does it not deploy with the configured environment? Are we trying to say that you can change that environment later if you want? Clarify or omit.